Header header
float32 accelerationX
float32 accelerationY
float32 accelerationZ

float32 angularRateX
float32 angularRateY
float32 angularRateZ

float64 latitude
float64 longitude
float32 altitude

float32 velocityX
float32 velocityY
float32 velocityZ

float32 heading
float32 pitch
float32 roll

float32 x
float32 y
float32 z

